content/datasets/_index.md

```diff
@@ -9,7 +9,9 @@ The Open Force Field Initiative makes a number of resources available to the com
 
 ---
 
-- [open-forcefield-data](https://github.com/openforcefield/open-forcefield-data): Curated datasets from the NIST ThermoML database to be used for force field parameterization.
+- [open-forcefield-data](https://github.com/openforcefield/open-forcefield-data): 
+  * Curated datasets from the NIST ThermoML database to be used for force field parameterization:
+    + 
```

Comment:
This extra space is rendering as a blank bullet point. You should remove it (one of the small joys of markdown)!

Reply:
Oh, I that's probably the extra space I added in my attempt to get bullet points to work in the first place. Thanks for catching it!